src/no/runsafe/toybox/command/Bazooka.java

```diff
@@ -37,19 +37,18 @@ public String OnExecute(IPlayer executor, IArgumentList parameters)
 			return null;
 		final Integer delay = parameters.getValue(DELAY);
 		final Float strength = parameters.getValue(STRENGTH);
-		final IEntity projectile = executor.Launch((RunsafeEntityType) parameters.getValue(ENTITY_TYPE));
-		if (projectile != null && delay != null && strength != null)
+		if (strength == null)
+			return "&cInvalid strength.";
+		if (strength > 255)

```

Comment:
You could handle this upstream with a `.min()`/`.max()` method on the argument specification. those could also have an optional second argument, which would be the error message to show if the value is out of range.